src/test/java/org/dice_research/opal/civet/metrics/DescriptionMetricImprovedTest.java

```diff
@@ -0,0 +1,93 @@
+	Model model;
+
+	@Before
+	public void setUp() throws Exception {
+		testdata = new TestData();
+		model = testdata.getModel(MODEL);
+	}
+
+	@Test
+	public void checkLessPosTags() throws Exception {
+
+		DescriptionMetricImproved metric = new DescriptionMetricImproved();
+		final String datasetUri = "http://projekt-opal.de/dataset/_mcloudde_stadtkrefeldgeodatenparkenstadtkrefeld";
+		Integer stars = metric.compute(model, datasetUri);
+		Assert.assertEquals("It contains less posTags", 2, stars.intValue());
+	}
+
+	@Test
+	public void noDescriptionProperty() throws Exception {
+
+		DescriptionMetricImproved metric = new DescriptionMetricImproved();
+		final String datasetUri = "http://projekt-opal.de/dataset/_mcloudde_tglichestationsmessungendermittlerenwindgeschwindigkeitinms";
+		Integer stars = metric.compute(model, datasetUri);
+		Assert.assertEquals("It contains no description property", 1, stars.intValue());
+	}
+
+	@Test
+	public void checkVeryLessPosTags() throws Exception {
+
+		DescriptionMetricImproved metric = new DescriptionMetricImproved();
+		final String TEST_EDP_ICE_DATASET = "http://projekt-opal.de/dataset/_mcloudde_stadtbonnstandortederbusparkpltze";
+		Integer stars = metric.compute(model, TEST_EDP_ICE_DATASET);
+		Assert.assertEquals("It contains very less posTags", 1, stars.intValue());
+	}
+
+	@Test
+	public void checkEmptyDescription() throws Exception {
+
+		DescriptionMetricImproved metric = new DescriptionMetricImproved();
+		final String TEST_EDP_ICE_DATASET = "http://projekt-opal.de/dataset/_mcloudde_standortederdauerzhlstellenradverkehrindsseldorf";
+		Integer stars = metric.compute(model, TEST_EDP_ICE_DATASET);
+		Assert.assertEquals("It contains empty description value", 1, stars.intValue());
+	}
+
+	@Test
+	public void checkAveragePosTags() throws Exception {
+
+		DescriptionMetricImproved metric = new DescriptionMetricImproved();
+		final String TEST_EDP_ICE_DATASET = "http://projekt-opal.de/dataset/_mcloudde_hvv-fahrplandatengtfsapril2018bisdezember2018";
+		Integer stars = metric.compute(model, TEST_EDP_ICE_DATASET);
+		Assert.assertEquals("It contains average posTags", 3, stars.intValue());
+	}
+
+	@Test
+	public void checkAboveAveragePosTags() throws Exception {
+
+		DescriptionMetricImproved metric = new DescriptionMetricImproved();
+		final String TEST_EDP_ICE_DATASET = "http://projekt-opal.de/dataset/_mcloudde_vieljhrlicherasterdesmittlerenvegetationsbeginnsindeutschland";
+		Integer stars = metric.compute(model, TEST_EDP_ICE_DATASET);
+		Assert.assertEquals("It contains above average posTags", 4, stars.intValue());
+	}
+
+	@Test
+	public void checkBestPosTags() throws Exception {
+
+		DescriptionMetricImproved metric = new DescriptionMetricImproved();
+		final String TEST_EDP_ICE_DATASET = "http://projekt-opal.de/dataset/_mcloudde_wettervorhersagemodellcosmo-de";
+		Integer stars = metric.compute(model, TEST_EDP_ICE_DATASET);
+		Assert.assertEquals("It contains best posTags", 5, stars.intValue());
```

Comment:
I replaced description with 20 words of lorem-ipsum. I still got 5 stars. This is the test case: https://file.io/faFTqweK
Could you please check ?